PSKoans/Koans/Constructs and Patterns/AboutSplatting.Koans.ps1

```diff
@@ -77,13 +80,22 @@ Describe 'Splatting' {
 
         It 'can be built from automatic hashtables' {
             $String = "Folders:__"
-            $Parameters = if ($String -match 'Folders:(?<Filter>[a-z ])$') {
+            $Parameters = if ($String -match 'Folders:(?<Filter>[_\d\w ]*)$') {
+                <#Matches is automaticed populated by the -match operater in the if statement
+                If $String = "Folders:00_TheBasics"
+                Then $Matches holds the following:
+                Name                           Value
+                ----                           -----
+                Filter                         00_TheBasics
+                0                              Folders:00_TheBasics
+                #>
                 $Matches.Remove(0) # Remove the 'whole' match and keep only the portions we asked for
-                $Matches.Clone()
+                $Matches.Clone() #Matches.Clone() allows for the remaining matches to be assigned to $Parameters
             }
             $Parameters.Add('Path', $PSKoansFolder)
             $Parameters.Add('Directory', $true)
-            (Get-ChildItem @Parameters).Count | Should -Be 2
+            #Now w
+            (Get-ChildItem @Parameters).Count | Should -Be __
```

Comment:
Partially missing comment? 🙂

Reply:
Yeah, literally no idea what comment I was going to leave here. So I have cleaned up that line.